classes/class-p4-campaign-exporter.php

```diff
@@ -101,7 +99,7 @@ public function add_import_button() {
 			?>
 			<script>
 				jQuery(function(){
-					jQuery("body.post-type-campaign .wrap .page-title-action").after('<a href="admin.php?import=wordpress" class="page-title-action"><?php _e( 'Import', 'planet4-master-theme-backend' ); ?></a>');
+					jQuery("body.post-type-campaign .wrap .page-title-action").after('<a href="admin.php?import=wordpress" class="page-title-action"><?php esc_attr_e( 'Import', 'planet4-master-theme-backend' ); ?></a>');
```

Comment:
Here you probably want `esc_html_e()` instead of `esc_attr_e()` since we are not echoing within an attribute.